frontend/language/src/nb.json

```diff
@@ -648,6 +648,7 @@
   "process_editor.not_found_process_error_message": "Det finnes ingen prosess definert innenfor BPMN, Du kan verifisere om prosessen finnes i BPMN-filen.",
   "process_editor.not_found_process_heading": "Ingen tilgjengelig prosess",
   "process_editor.save": "Lagre",
+  "process_editor.saved_successfully": "Bpmn prosessen ble lagret",
```

Comment:
Her må det være bindestrek: `Bpmn-prosessen ble lagret.`

Reply:
@TomasEng fixed in this: https://github.com/Altinn/altinn-studio/pull/11866